lib/arrays.md

```diff
@@ -5,44 +5,52 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An array is composed of values stored in cells. It is defined by brackets in Ruby.
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+An array and a string can form the same block of text, but because an array can
+store words of the text into its cells. Strings are composed of single values.
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
 
-Replace me with your response...
+If an array has n elements then to call the nth element the command array_name[n].
```

Comment:
In Ruby, if I have an array with three elements `arr = [1, 2, 3]` and I try to call the third element with `arr[3]`, my return is blank/nil. 

The `[n]` in your answer represents the _index_ of an Array. What is the starting index for an Array?